src/index.ts

```diff
@@ -34,6 +35,50 @@ const compileTemplate = async (
   });
 };
 
+const processRust = async (
+  destinationDirectoryName: string,
+  generatorOptions: IGeneratorOptions,
+) => {
+  const cargoTOMLPath = path.join(destinationDirectoryName, "Cargo.toml");
+  const fileContents = await readFile(cargoTOMLPath);
+  const cargoTOML = TOML.parse(fileContents.toString());
+  const updatedCargo = TOML.stringify({
+    ...cargoTOML,
+    package: {
```

Comment:
Looks like you’re rewriting the whole package section here, not just updating the name. That trims the edition flag, which would cause the failure we see on CI.